docs/articles/rag-application-communication-system.md

```diff
@@ -0,0 +1,164 @@
+
+After designing multiple RAG systems over the past year, I’ve come to believe that RAG signals a more radical transformation: the integration of LLMs in new forms of flexible expert systems. Effective RAG means ingesting even heterogeneous, badly formatted data sources and contextualizing new knowledge on the fly. RAG applications should aspire to become a communication system, taking the best features out of language models (adaptability, universal content translation) but also moderating their shortcomings (alignment mismatch, hallucinations, lack of explainability).
+
+As Yan et al. accurately [called it](https://applied-llms.org/#dont-forget-keyword-search-use-it-as-a-baseline-and-in-hybrid-search): “Models are likely to be the least durable component in the system”. They can be swapped, combined or disabled. What this means for the months and the years to come is that, rather than being replaced by some disruptive LLM technology like long context, RAG will eat LLMs. Embedding search is largely repurposed as an RAG technique. These days, I am mostly fine-tuning with RAG in mind. I recently co-founded PleIAs to get even one step further by pre-training a new generation of language models for RAG end use, and increased support for document processing use cases.
+
+## 1. Language models are few-doc learners.
+RAG was not supposed to exist. With pre-training scaling up, LLMs have been largely conceived as “interpolated datasets” holding in memory the collective knowledge of the Internet. The inaugural RAG system was still a form of interpolated dataset: a clever model design associating a seq2seq model with a dense layer trained on Wikipedia ((Lewis et al., 2020)[https://proceedings.neurips.cc/paper/2020/file/6b493230205f780e1bc26945df7481e5-Paper.pdf]).
+
+Interpolated datasets ran into a range of structural issues that proceed from the core nature of LLMs (a model trained at one time on a finite set of data):
+* **Specialization**: many professional resources are unlikely to be featured in pre-training data or with a very light exposure that result in high hallucination rates.
+* **Updatability**: a database can be continuously maintained in real time while LLMs are limited by their cutoff date.
+* **Explainability**: separating the search/memory part does circumvent to some extent the “black box” nature of LLMs.
+
+Instead, reformulation of few-document turned out to be one the most practical use cases for LLMs. The key properties were already built by design in word embeddings nearly a decade ago and have developed ever since. Cross-lingual transfer is surprisingly efficient: many models primarily trained in English can be successfully trained on downstream tasks in other languages. This extends to other forms of language transfer: from formal to informal, from standards to dialects, from generalist to specialized terminologies.
```

Comment:
instead of interpolated datasets? instead of what?

Reply:
Suggestion: "For all theses reasons, grounding LLM on existing data at inference time turned out to be the most practical use cases for LLMs. This approach takes advantage of one of the most valuable property of LLMs: rephrasing and reformulating an existing text in a new style or for a different audience." => then Cross-Lingual or just cut off the rest.